### PR TITLE
[DPM-905] send newplayer on sign-up

### PR DESCRIPTION
**Pull request by georgiypetrov**

### CODE REVIEW

File: auth/usecase/usecase.go [hunk 1]
```diff
@@ -110,6 +113,12 @@ func (a *AuthUseCase) SignUp(ctx context.Context, user *models.User) (string, st
 			log.Debug().Msgf("User add error: %s", err.Error())
 			return "", "", err
 		}
+
+		go func() {
+			if err := a.contractUC.SendNewPlayerToCasino(ctx, user.AccountName, casinoName); err != nil {
```
Comment (justefg) [inline auth/usecase/usecase.go:118]: `SendBonusToNewPlayer` ?

File: auth/usecase/usecase.go [hunk 2]
```diff
@@ -110,6 +113,12 @@ func (a *AuthUseCase) SignUp(ctx context.Context, user *models.User) (string, st
...
+				log.Debug().Msgf("Send new player to casino error: %s", err.Error())
```
Comment (justefg) [inline auth/usecase/usecase.go:119]: warning would be better, i think?

File: server/http.go [hunk 3]
```diff
@@ -90,7 +94,7 @@ func authHandler(app *App, w http.ResponseWriter, r *http.Request) {
 		}
 		user.AffiliateID = req.AffiliateID
 	}
-	refreshToken, accessToken, err := app.useCases.Auth.SignUp(context.Background(), user)
+	refreshToken, accessToken, err := app.useCases.Auth.SignUp(context.Background(), user, casinoName)
```
Comment (justefg) [inline server/http.go:97]: if it's not development mode how do we get `casinoName` for a user?

File: server/http.go [hunk 4]
```diff
@@ -64,13 +64,17 @@ func respondWithJSON(w http.ResponseWriter, code int, payload interface{}) {
...
 	if app.developmentMode {
 		user = &models.User{
 			AccountName: "testuserever",
 			Email:       "test@user.ever",
 			AffiliateID: "afff",
 		}
+		casinoName = "casino"
 	} else {
 		var req AuthRequest
 		err := json.NewDecoder(r.Body).Decode(&req)
```
Comment (justefg) [inline server/http.go:80]: frontend should be deployed first, right?

File: auth/usecase/usecase.go (hunk 1 above)
Comment (georgiypetrov) [reply auth/usecase/usecase.go:118]: ok

File: auth/usecase/usecase.go (hunk 2 above)
Comment (georgiypetrov) [reply auth/usecase/usecase.go:119]: ok

**Review by justefg — APPROVED** [review]

**Pull request merged**